db/migrate/20171110195947_devise_token_auth_create_users.rb

```diff
@@ -0,0 +1,28 @@
+class DeviseTokenAuthCreateUsers < ActiveRecord::Migration[5.1]
+  def change
+    change_table(:users) do |t|
+      ## Required
+      t.string :provider, :null => false, :default => "email"
+      t.string :uid, :null => false, :default => ""
+
+      ## Confirmable
+      # t.string   :confirmation_token
+      # t.datetime :confirmed_at
```

Comment:
While documentation lines are ok, I'd remove lines like these in migrations (they don't add much, if we want to add confirmable later we will just create another migration with those fields).